TrackPropagation/RungeKutta/test/rkTest_cfg.py

```diff
@@ -20,16 +20,17 @@
 
 ### Full field map, static configuration for each field value
 #process.load("Configuration.StandardSequences.MagneticField_20T_cff")
-
 #process.load("Configuration.StandardSequences.MagneticField_30T_cff")
-
 #process.load("Configuration.StandardSequences.MagneticField_35T_cff")
-
 #process.load("Configuration.StandardSequences.MagneticField_38T_cff")
-
 #process.load("Configuration.StandardSequences.MagneticField_40T_cff")
 
-process.load("MagneticField.Engine.volumeBasedMagneticField_090322_2pi_scaled_cfi")
+### Configuration to select map based on recorded current in the DB
+process.load("Configuration.StandardSequences.FrontierConditions_GlobalTag_cff")
+from Configuration.AlCa.GlobalTag import GlobalTag
+process.GlobalTag = GlobalTag(process.GlobalTag, 'auto:phase2_realistic', '')
```

Comment:
is it really necessary to use the phase2 GT here?
I know that "any random would be OK", but may be better to chose something less possibly in flux.
run1 may be more stable for this purpose.